explainaboard/metrics/nlg_meta_evaluation.py

```diff
@@ -188,21 +188,20 @@ def _count(self, score: list, config: Optional[MetricConfig] = None):
         num = 0
         for i in range(1, len(score)):
             for j in range(0, i):
-                if (
-                    abs(score[i][0] - score[j][0]) < config.threshold
-                    or abs(score[i][0] - score[j][0]) == 0
-                ):
+                manual_diff = score[i][0] - score[j][0]
+                system_diff = score[i][1] - score[j][1]
+                if abs(manual_diff) < config.threshold or abs(manual_diff) == 0:
```

Comment:
I noticed that the following is much simpler:
```python
if manual_diff >= config.threshold:
   ...  # i is better than j
elif manual_diff <= -config.threshold:
   ...  # i is worse than j
#that's all
```

Reply:
Thanks. Fixed.